bpdv3/Assets/Scripts/Managers/TestUIManager.cs

```diff
@@ -0,0 +1,50 @@
+﻿namespace Assets.Scripts.Managers 
+{
+    using TMPro;
+    using UnityEngine;
+
+    public class TestUIManager : MonoBehaviour
+    {
+        #region Private & Const Variables
+
+        private float _timer;
+
+        #endregion
+
+        #region Public & Protected Variables
+
+        public TextMeshProUGUI TimerUiElement;
+
+        #endregion
+
+        #region Constructors
+
+        #endregion
+
+        #region Private Methods
+
+        private void Update()
+        {
+            IncrementTimer();
+        }
+
+        private void IncrementTimer()
+        {
+            _timer += Time.deltaTime;
+
+            if (TimerUiElement?.text == null) 
+            {
+                return;
+            }
+
+            TimerUiElement.text = _timer.ToString("00");
+        }
+
+        #endregion
+
+        #region Public & Protected Methods      
+
+        #endregion
+    }
+}
+
```

Comment:
I like the structure u use where u categories sections of the code for like private and consts, public and protected etc. Eventually we can have the project using one style of code. I'm happy to use the way you code as I think I have plenty I can learn from you. I'll try pick up what I can without being pest thou :).

Reply:
I think when im able to get a job as a coder, ill be able to learn a lot there. But whilst i'm still looking for work and working on my portfolio, I will just have to make do with what i know now or pick up along the way. l'm looking for my first coding job since completing my course last year.